views/home.html

```diff
@@ -0,0 +1,44 @@
+% include('base.html', title="PyJudge")
+<body>
+    <div class="container text-center">
+        <h1>
+            PyJudge
+        </h1>
+    </div>
+    <div class="container">
+        <ul class="nav nav-tabs">
+            <li class="nav-item">
+                <a class="nav-link active" data-toggle="tab" href="#login">Login</a>
+            </li>
+            <li class="nav-item">
+                <a class="nav-link" data-toggle="tab" href="#register">Register</a>
+            </li>
+        </ul>
+        <div class="tab-content">
+            <div id="login" class="tab-pane container active">
+                <form name="login" method="post" action="/login">
+                    <label>Username:</label>
+                    <input type="text" name="username" required /><br />
+                    <label>Password</label>
+                    <input type="password" name="password" required /><br />
+                    <input class="btn btn-primary" type="submit" value="Login" />
+                </form>
+            </div>
+            <div id="register" class="tab-pane container fade">
+                <form name="register" method="post" action="/register">
+                    <label>Username:</label>
+                    <input type="text" name="username" required /><br />
+                    <label>First name:</label>
+                    <input type="text" name="firstname" required /><br />
+                    <label>Last name:</label>
+                    <input type="text" name="lastname" /><br />
+                    <label>Password:</label>
+                    <input type="password" name="password" required /><br />
+                    <label>Confirm Password:</label>
```

Comment:
let's get rid of this part? Instead just show the password in plain text when registering and the ability to toggle it (if you're feeling up to it. I'm ok with a plaintext password)

Reply:
I would prefer hidden instead of plaintext as it is a password field, your call.